test/problems.jl

```diff
@@ -0,0 +1,56 @@
+# ---------------------------------------------------------
```

Comment:
the functions in this file `problems.jl` now return the tuple (initial value problem, analytic solution)